src/util/random-string.ts

```diff
@@ -1,4 +1,15 @@
 export const createRandomString = () => {
+    if (
+        !('crypto' in window) ||
+        typeof window.crypto?.randomUUID !== 'function'
+    ) {
+        return legacyCreateRandomString();
```

Comment:
Is it expected that crypto.randomUUID won't be available in many cases? 

If so, why use it at all? Why not just use the legacyCreateRandomString function?

Reply:
I suppose there is full support on all major browsers since about the end of 2021, although better safe than sorry. Still, the test suite still falls back on the legacy one (maybe it's not running a "secure context", I'm not sure)